features/ouders.feature

```diff
@@ -10,7 +10,7 @@ Rule: De ouder in categorie 2 heeft ouderAanduiding "1" en de ouder in categorie
 
   @gba
```

Comment:
De @gba tags kunnen worden verwijderd, omdat we hebben afgesproken end to end te testen

Reply:
Ga ik in een nieuwe PR aanpassen.